tez-dag/src/main/java/org/apache/tez/dag/app/dag/impl/OutputCommitterContextImpl.java

```diff
@@ -34,14 +34,16 @@ public class OutputCommitterContextImpl implements OutputCommitterContext {
   private final String dagName;
   private final String vertexName;
   private final int vertexIdx;
+  private final int dagIdentifier;
   private final RootInputLeafOutput<OutputDescriptor, OutputCommitterDescriptor> output;
 
   public OutputCommitterContextImpl(ApplicationId applicationId,
       int dagAttemptNumber,
       String dagName,
       String vertexName,
       RootInputLeafOutput<OutputDescriptor, OutputCommitterDescriptor> output,
-      int vertexIdx) {
+      int vertexIdx,
+      int dagIdentifier) {
```

Comment:
if we're already changing the signature, what about another order that looks better?
is this context used somewhere else other than tez code? I guess it's not, we can use whatever we want, what about:
```
dagAttemptNumber
dagName
vertexName
dagId
vertexId
output
```

Reply:
Refactored.